src/components/Hero/ContentMedia/index.tsx

```diff
@@ -12,47 +13,44 @@ import { Page } from '@root/payload-types'
 import classes from './index.module.scss'
 
 export const ContentMediaHero: React.FC<
-  Pick<Page['hero'], 'richText' | 'media' | 'mediaWidth' | 'links'> & {
+  Pick<Page['hero'], 'richText' | 'media' | 'links' | 'sidebarContent'> & {
     breadcrumbs?: Page['breadcrumbs']
   }
-> = ({ richText, media, mediaWidth, breadcrumbs, links }) => {
+> = ({ richText, media, links, sidebarContent }) => {
   return (
     <Gutter>
-      <div
-        className={[mediaWidth === 'wide' ? classes.wideGrid : classes.normalGrid, 'grid']
-          .filter(Boolean)
-          .join(' ')}
-      >
-        <div
-          className={[`cols-${mediaWidth === 'wide' ? 6 : 8}`, 'start-1 cols-m-8']
-            .filter(Boolean)
-            .join(' ')}
-        >
-          <Breadcrumbs items={breadcrumbs} />
-          <RichText content={richText} />
-          {Array.isArray(links) &&
-            links.map(({ link }, i) => {
-              return <CMSLink key={i} {...link} className={classes.link} />
-            })}
+      <div className={[classes.wrapper, 'grid'].filter(Boolean).join(' ')}>
+        <BackgroundGrid ignoreGutter />
+        <div className={[classes.sidebar, `cols-4`, 'cols-m-8 start-1'].filter(Boolean).join(' ')}>
+          <RichText content={richText} className={[classes.richText].filter(Boolean).join(' ')} />
+
+          <div className={[classes.linksWrapper].filter(Boolean).join(' ')}>
+            <RichText
+              content={sidebarContent}
```

Comment:
@paulpopus When working through the rest of the `Heros` other than the `home` one - I would look to see which ones use this `sidebarContent` prop

Looks like its the `ContentMedia` one and `Default` one.

If your plan is to update the `default` style as well, I would consider changing the name of this prop to something else. `sidebarContent` doesn't really make sense in this ContentMedia design, it's more-so just `content`